solr/core/src/test-files/solr/collection1/conf/schema.xml

```diff
@@ -471,12 +471,13 @@
     </analyzer>
   </fieldType>
   <!-- 
-    Example of using PathHierarchyTokenizerFactory at query time, so
-    queries for paths match documents at that path, or in ancestor paths
+    Example of using PathHierarchyTokenizerFactory for ancestor queries.
+    Since Lucene 10 produces sequential tokens, we need to store paths at index time
+    to match against sequential query tokens properly.
   -->
   <fieldType name="ancestor_path" class="solr.TextField">
     <analyzer type="index">
-      <tokenizer class="solr.KeywordTokenizerFactory"/>
+      <tokenizer class="solr.PathHierarchyTokenizerFactory" delimiter="/"/>
     </analyzer>
     <analyzer type="query">
       <tokenizer class="solr.PathHierarchyTokenizerFactory" delimiter="/"/>
```

Comment:
minor/subjective: could combine the two `analyzer` elements since they are now identical